features/step_definitions/operators.rb

```diff
@@ -49,3 +49,9 @@
   })
   step %Q/the step should succeed/
 end
+
+# Get the Major.Minor cluster version
+Given /^the major.minor version of the cluster is stored in the#{OPT_SYM} clipboard$/ do | cb_name |
+  cb_name = 'operator_channel_name' unless cb_name
+  cb[cb_name] = cluster_version('version').channel.split('stable-')[1]
```

Comment:
@jianzhangbjz are you sure `stable-` is always there?

Reply:
@pruan-rht Thanks! I guess `-` always there. Changed it to `-`.